baggers.js

```diff
@@ -16188,6 +16188,58 @@ var data = {
         twitter: 'cmaneu',
         mail: 'hello@maneu.net',
       },
+    },
+	{
+      since: '2019-04-06',
+      name: 'Mimoun Kissi',
+      bio:
+        'Coach qualité, QA Engineer, passioné de tests et qualités logiciels',
+      picture: 'https://www.dropbox.com/s/5oxgukkq92304wp/IMG_0551.jpg?dl=0',
+      websites: [
+        {
+          name: 'Github',
+          url: 'https://github.com/MimounKissi',
+        },
+        {
+          name: 'Biography',
+          url: 'https://mimoun-kissi.carrd.co',
+        },
+        {
+          name: 'Calendly',
```

Comment:
This is not part of the standard names we documented in the readme. Could you fix it?